### DISCUSSION TITLE
My pydiode environment status has been showing 'busy' and code has refused to run

### DISCUSSION THREAD

**Discussion by jaytriggers** [pyodide/pyodide#5042] [General]
Since about 24 hours ago when i used a function - 'iterable.product' to do some complex calculation my piodide has been processing slow or not at all, it took around 5-10 minutes to get the output for the code I ran with iterable.product and from then one it has either taken almost 30 minutes for a simple library importation or completely hung with the 'busy' status. what do I do please?

**Comment by ryanking13** [comment]
Well, you'll first need to profile which part of the code takes much time. Then we can help you debug the issue.
